src/libs/actions/Policy/DistanceRate.ts

```diff
@@ -257,7 +261,11 @@ function clearPolicyDistanceRatesErrorFields(policyID: string, customUnitID: str
     });
 }
 
-function clearDeleteDistanceRateError(policyID: string, customUnitID: string, rateID: string) {
+function clearDeleteDistanceRateError(policyID: string, customUnitID: string | undefined, rateID: string) {
+    if (!customUnitID) {
+        return;
+    }
+
```

Comment:
if this is true, do we catch this error in any way? Should we be calling clearDeleteDistanceRateError without customUnitID?